src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.cpp

```diff
@@ -324,6 +325,90 @@ void jit_exp_emitter::register_table_entries() {
     push_arg_entry_of("exponent_bias", 0x0000007f);
 }
 
+/// FLOOR ///
+jit_floor_emitter::jit_floor_emitter(jit_generator* host, cpu_isa_t host_isa, const element::Type exec_prc)
+    : jit_emitter(host, host_isa, exec_prc) {
+        prepare_table();
+    }
+
+jit_floor_emitter::jit_floor_emitter(ov::intel_cpu::riscv64::jit_generator* host, ov::intel_cpu::riscv64::cpu_isa_t host_isa, const std::shared_ptr<ov::Node>& node) :
+    jit_emitter(host, host_isa, get_arithmetic_binary_exec_precision(node)) {
+        prepare_table();
+}
+
+size_t jit_floor_emitter::get_inputs_num() const {
+    return 1;
+}
+
+size_t jit_floor_emitter::aux_vecs_count() const {
+    return 2; 
+}
+
+size_t jit_floor_emitter::aux_fp_gprs_count() const {
+    return 2; 
+}
+
+void jit_floor_emitter::emit_impl(const std::vector<size_t>& in_vec_idxs, const std::vector<size_t>& out_vec_idxs) const {
+    if (host_isa_ == ov::intel_cpu::riscv64::cpu_isa_t::gv) {
+        emit_isa<ov::intel_cpu::riscv64::cpu_isa_t::gv>(in_vec_idxs, out_vec_idxs);
+    } else {
+        OPENVINO_THROW("Can't create jit eltwise kernel for FLOOR");
+    }
+}
+void jit_floor_emitter::register_table_entries() {
+
+    push_arg_entry_of("zero", 0x00000000);    
```

Comment:
On RISC-V 64 the general-purpose register `zero` always contain `0x0` value (zero). So if you need to work with zero, you don't need to push `0x0` value to data section - you can just use the register `zero`.

[Example](https://github.com/openvinotoolkit/openvino/blob/28bc1fa8d794fea34acbb34f67bcc16142800053/src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.cpp#L463) in `jit_relu_emitter` where we move `zero` to floating-point register for the further work (formula of `Relu` is `y = max(x, 0)`)